src/app/model/User.ts

```diff
@@ -28,10 +35,34 @@ class User extends BaseModel {
   VolunteerStatus = VolunteerStatus;
 
   /**
-   * Given a displayName returns the first matching user object
-   * @param {string} displayName : displayName of user
+   * Given a mission object creates a new mission in firestore
+   * returns the new mission id
+   * @param {object} mission
    * @return {string}
    */
+  async createMission(mission: MissionInterface): Promise<string> {
+    const missionId = uuidV4(); //generate mission id
+    const collection = this.getCollection("missions");
+
+    //Add mission id to mission object
+    mission.id = missionId;
+
+    //save mission in firestore
+    try {
+      await collection.doc(missionId).set(mission);
```

Comment:
so yeh here is where we would want to use `load`

Reply:
Also any particular reason this was moved to `User` instead of `Mission`?

Reply:
> so yeh here is where we would want to use `load`
 
The `load` call is outside of this function in MissionCreate.jsx (If that's what you mean). Moved it because @mat10tng mentioned he wants the data to be sent to backend in User.createMission in #247.

Reply:
for the sake of sanity when create object, we should do an extra load here, just in case ppl mutate the object outside without our knowing

Reply:
Yeah I guess that makes a lot more sense!